src/worker/indexedDBAccess.ts

```diff
@@ -0,0 +1,43 @@
+import Dexie from "dexie";
+
+const DB_NAME = "dictionary_sources";
+const VERSION = 1;
+
+/**
+ * files Table Scehma
+ * +-------+
+ * | id    |
+ * +-------+
+ * | name  |
+ * +-------+
+ * | file  |
+ * +-------+
+ */
+const FILES_TABLE_NAME = "files";
+const FILES_TABLE_SCHEMA = "++id, name, file";
+
+interface IFile {
+  id?: number;
+  name: string;
+  file: File;
+}
+
+class IndexDBAccess extends Dexie {
+  files: Dexie.Table<IFile, number>;
+
+  constructor() {
+    super(DB_NAME);
+    this.version(VERSION).stores({ files: FILES_TABLE_SCHEMA });
+    this.files = this.table(FILES_TABLE_NAME);
```

Comment:
Doesn't Dexie export all tables as properties like this anyway? e.g., can't you do `db.files.where(...)` or `db.files.put(...)` etc?

Reply:
Ignore this! I got it :)